### PR TITLE
Backport master and sync CCM/CSI annotations

### PR DESCRIPTION
**Pull request by chrisho**
Backport master and sync CCM/CSI annotations

```yaml
annotations:
  catalog.cattle.io/certified: rancher
  catalog.cattle.io/namespace: kube-system
  catalog.cattle.io/release-name: harvester-cloud-provider/harvester-csi-driver
  catalog.cattle.io/os: linux
  catalog.cattle.io/permits-os: linux
  catalog.cattle.io/rancher-version: ">= 2.6.1-0 < 2.7.0-0"
  catalog.cattle.io/ui-component: harvester-cloud-provider/harvester-csi-driver
  catalog.cattle.io/display-name: Harvester Cloud Provider/Harvester CSI Driver
  catalog.cattle.io/kube-version: ">= 1.18.0-0 < 1.25.0-0"
```

### CODE REVIEW

File: charts/harvester-cloud-provider/values.yaml [hunk 1]
```diff
@@ -6,9 +6,9 @@ replicasCount: 1
 
 image:
   repository: rancher/harvester-cloud-provider
-  pullPolicy: Always
```
Comment (yaocw2020) [inline charts/harvester-cloud-provider/values.yaml:9]: `Always` is OK for `master-head` verison.

Comment (chrisho) [reply charts/harvester-cloud-provider/values.yaml:9]: Now CSI is `IfNotPresent`, modify to `Always` too?

Comment (chrisho) [reply charts/harvester-cloud-provider/values.yaml:9]: Now CSI is `IfNotPresent`, modify to `Always` too?

Comment (yaocw2020) [reply charts/harvester-cloud-provider/values.yaml:9]: Yes, it makes sense.

**Comment by chrisho** [comment]
Outdated

**Pull request closed without merging**